### ISSUE TITLE
MEME

### ISSUE THREAD

**Issue by shinysyk** [ubccr/software-layer#269]
Hi. My name is Yungki Park. I am the course director of BCH519. I request that MEME is installed at CCR. Its installation instructions are found at the following website.
https://meme-suite.org/meme/doc/install.html?man_type=web

MEME is the most widely used tool for motif analysis. It is also essential for the teaching of bioinformatics (BCH519).

Thanks for your help. If there is anything that I can help for this installation, please let me know.

**Comment by tonykew** [comment]
Test build of MEME version 5.4.1 complete - more info to follow

**Comment by tonykew** [comment]
MEME version 5.4.1 has been published to the ccrsoft/2023.01 software release

```
$ module spider meme
----------------------------------------------------------------------------
  meme: meme/5.4.1
----------------------------------------------------------------------------
    Description:
      The MEME Suite allows you to: * discover motifs using MEME, DREME
      (DNA only) or GLAM2 on groups of related DNA or protein sequences, *
      search sequence databases with motifs using MAST, FIMO, MCAST or
      GLAM2SCAN, * compare a motif to all motifs in a database of motifs, *
      associate motifs with Gene Ontology terms via their putative target
      genes, and * analyse motif enrichment using SpaMo or CentriMo.


    You will need to load all module(s) on any one of the lines below before the "meme/5.4.1" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
$ module load gcc/11.2.0 openmpi/4.1.1 meme/5.4.1
$
```

The MEME module includes multiple binaries:

```
$ ls ${EBROOTMEME}/bin
ame       dust   glam2scan  mcast      momo   spamo   tomtom
centrimo  fimo   gomo       meme       purge  streme  xstreme
dreme     glam2  mast       meme-chip  sea    tgene
$ 
```

e.g.

```
$ meme -h
 Usage:	meme	<dataset> [optional arguments]

	<dataset> 		file containing sequences in FASTA format
	[-h]			print this message
	[-o <output dir>]	name of directory for output files
				will not replace existing directory
[...]
$ 
```
